### ISSUE TITLE
Fix Search Page sizing on Mobile

### ISSUE THREAD

**Issue by PrivatePasta** [ronggang/PT-Plugin-Plus#917]
Device: POCO X3 Pro
Resolution: 2400x1080
Browser: Kiwi Browser
PTPP Version: v.1.5.2

**Is your feature request related to a problem? Please describe.**
The Search Page on mobile is very congested because of the multiple columns. Especially the comments column which does not need to be so long. Because of this, important information like the full torrent name is hidden (group name is obviously important) and the only way to fix it is by going to each page to check the full name.

**Describe the solution you'd like**
Give us the option to add or remove columns in each page so that we can have it fit every device.

**Describe alternatives you've considered**
Switching to landscape mode. Doesn't fix the issue.

**Additional context**
This is what it look like when trying to search anything. As you can see the trailing part of the torrent title is hidden and I'd like to change that <br>
![Screenshot_20211022-005911](https://cdn.upload.systems/uploads/iMNEXJBt.png)<br>
This is with "Desktop Mode" on because PTPP is genuinely unusable without it.

**Comment by PrivatePasta** [comment]
Are you sure? I'm on version 1.5.2.3f9c990 and it looks like this for me. I followed the Developers Instructions and used dev branch. 

![Screenshot_20211030-020628](https://user-images.githubusercontent.com/63346955/139498913-de7f1bfc-c20f-4483-baf2-1e73cad96cfd.png)

**Comment by PrivatePasta** [comment]
Tried again with the latest version - 1.5.2.139a742 - and it's the same as above. My device details, if it matters, are in the first comment of this issue.

**Comment by ted423** [comment]
@PrivatePasta So, use CMT?

**Comment by PrivatePasta** [comment]
CMT?

**Comment by Rhilip** [comment]
> CMT?

short for word `comment`, but I don't think it's necessary. 

The main point for this issue is too much columns in table, 
and the column headers are not editable.
At now, we add css `text-overflow: ellipsis;` and `title` attribute for title columns.
it works well in desktop but not mobile,
but the browser in mobile not listen to event mouseover.

**Comment by PrivatePasta** [comment]
@Rhilip yeah that was my suggestion. You could hard-code the comment column to be of shorter width. But a better option would be to give users the option as to which columns are shown.

**Comment by ted423** [comment]
@PrivatePasta not too many option. Have you tried horizontal screen

**Comment by PrivatePasta** [comment]
Yeah, I mentioned in my first comment on the issue. Landscape doesn't change anything cus the column stays the same size.

**Comment by Rhilip** [comment]
Since the master and dev branch is most like the stable status, 
and The ui framework is too old ( Vuetify v1.5 )

So I don't want to do this now, and I want to fix this issue in PTPP v2 ( the [next](https://github.com/ronggang/PT-Plugin-Plus/tree/next) branch )
